docs: reorder add-ons index (#408)
CleverCloud/documentation: content/doc/addons/matomo.md

type: docs
---

> Matomo is an open-source web analytics solution which gives you full ownership of your data. It's a [GDPR compliant](https://matomo.org/gdpr-analytics) alternative to Google Analytics. You can learn more about Matomo on [their website](https://matomo.org). Matomo on Clever Cloud will allow your marketing team to effortlessly setup a tailored web analytics solution, that can be adjusted to your needs and workloads.

Reviewer: ```suggestion
Matomo is an open source web analytics solution which gives you full ownership of your data. It's a [GDPR compliant](https://matomo.org/gdpr-analytics) alternative to Google Analytics. You can learn more about Matomo on [their website](https://matomo.org). Matomo on Clever Cloud allows your marketing team to setup a tailored web analytics solution, that adapts to your specific needs and workloads.
```

{{< callout type="info" >}}
**Matomo is in Beta testing phase:** your insights and suggestions are crucial in shaping the future of this platform.

> To share your feedback, please visit us at [our community page](https://github.com/CleverCloud/Community/discussions/categories/matomo).

Reviewer: ```suggestion
To share your feedback, visit us at [this documentation community forum](https://github.com/CleverCloud/Community/discussions/categories/matomo).
```

> Thank you for being a part of our journey towards innovation and improvement!

Reviewer: Accepted for callouts about beta phases
